sdk/src/main/java/com/qonversion/android/sdk/listeners/QonversionCallback.kt

```diff
@@ -29,6 +29,11 @@ interface QonversionRemoteConfigCallback {
    fun onError(error: QonversionError)
 }
 
+interface QonversionExperimentAttachCallback {
```

Comment:
QonversionExperimentAttachCallback is used both for attach and detach. Maybe something unrelated to experiments, like `QonversionSimpleCallback` as it can be used in other places, where no response body is needed